### PR TITLE
feat: return the growth rate together with the simulations results

### PR DESCRIPTION
**Pull request by sgalkina**

### CODE REVIEW

File: iloop_to_model/iloop_to_model.py [hunk 1]
```diff
@@ -147,6 +147,7 @@ def extract_measurements_for_phase(scalars_for_samples):
 MEASUREMENTS = 'measurements'
 REACTIONS = 'reactions-knockout'
 MODEL = 'model'
+GROWTH_RATE = 'growth-rate'
```
Comment (matyasfodor) [inline iloop_to_model/iloop_to_model.py:150]: I'd prefer underscores or camelcase (although not very Python-y). In js you can't access property names containing `-` with the dot notation, making it less convenient. WDYT? Also mentioning @kvikshaug

Comment (kvikshaug) [reply iloop_to_model/iloop_to_model.py:150]: Note that this doesn't reach JS; the response from venom **is** camelcased in the end. This is used in the request to the `model` service.
However I'm all for standardizing on camelcase as part of a larger task.

File: iloop_to_model/app.py [hunk 2]
```diff
@@ -253,6 +253,8 @@ class Meta:
         return ModelsMessage(response={k: ModelMessage(
             model=JSONValue(v['model']),
             model_id=v['model_id'],
+            # if the growth rate is 0.0, venom detects the field as undefined
+            growth_rate=v['growth-rate'] + 1e-106,
```
Comment (kvikshaug) [inline iloop_to_model/app.py:257]: Creative fix 😄
I don't see why venom should skip serializing zero numbers (which was implemented [here](https://github.com/biosustain/venom/commit/0e147eb772aed0363ee20778c342589aadf6925d)). So FYI I suggested to [fix it in venom](https://github.com/biosustain/venom/pull/27), but if it's not accepted or takes too long then this trick works fine.

**Review by kvikshaug — APPROVED** [review]

File: iloop_to_model/iloop_to_model.py (hunk 1 above)
Comment (matyasfodor) [reply iloop_to_model/iloop_to_model.py:150]: I see, then no need to change this.

**Review by matyasfodor — APPROVED** [review]

**Pull request merged**